### ISSUE TITLE
server-pgsql always tries to create schema

### ISSUE THREAD

**Issue by sboschman** [zabbix/zabbix-docker#45]
Table exists check is for table public.dbversion:
https://github.com/zabbix/zabbix-docker/blob/3.2/server-pgsql/alpine/run_zabbix_component.sh#L339

If the table exists the version is retrieved from a table in the wrong schema (zabbix.dbversion):
https://github.com/zabbix/zabbix-docker/blob/3.2/server-pgsql/alpine/run_zabbix_component.sh#L344
'zabbix' being the default database name; but in the query the database name is used as schema name.

Also the comment https://github.com/zabbix/zabbix-docker/blob/3.2/server-pgsql/alpine/run_zabbix_component.sh#L348 is not correct, the schema.sql create script uses the default schema 'public', not a schema with the same name as de database.

Basically the database name is sometimes misused as schema name.

Upgrading from e.g. 2.4 is not straighforward like this. The database is 2.4. Because the version detection fails zabbix-server tries to create a fresh database. Apart from the already exists errors new 3.x tables are created. Afterwards the database upgrade is triggered and fails on already existing tables.

A workaround is to add a pg view 'zabbix.dbversion' to the database, which is 'select * from public.dbversion' before you start the zabbix-server container for the first time.

**Comment by dotneft** [comment]
solved. Thank you!
